web/bower.json

```diff
@@ -24,7 +24,8 @@
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "d3": "3.5.5",
     "d3-tip": "^0.6.7",
-    "moment": "^2.10.6"
+    "moment": "^2.10.6",
+    "jquery": "^2.1.4"
```

Comment:
I think angular can use jQuery rather than jqLite, but you're right that it is not necessary - angular-ui-bootstrap's implementation doesn't require jQuery like the datepicker otherwise would